frontend/src/widgets/HorizontalTableWidget.scss

```diff
@@ -115,4 +103,33 @@
 
 .smarthub-horizontal-table-widget tfoot td {
   font-weight: bold;
-}
\ No newline at end of file
+}
+
+// First footer cell sticky.
+.smarthub-horizontal-table-widget tfoot td:first-child {
+  left: 0;
+  position: sticky;
+  z-index: 1;
+  min-width: 64px;
+}
+
+// Second footer cell sticky.
+.smarthub-horizontal-table-widget tfoot td:nth-child(2) {
+  position: sticky;
+  z-index: 1;
+  max-width: 330px;
+  left: 44px;
+}
```

Comment:
Can we use a class to select these instead of :first-child, nth-child?

Reply:
Its a map will leave as is for now.

Reply:
I put this back so the border only shows in larger views.